lib/open_weather/models/station.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module OpenWeather
+  module Models
+    class Station < Model
+      # internal identifier for the station
```

Comment:
Be consistent, put the comment after the code line.

Reply:
My bad, I was trying to see if it could work with both 'id' and 'ID' and thought the line was too long

Reply:
You can probably alias it if you really want to.